# MBDP Metagenomics 2024

1. [General introduction](#general-introduction)
2. [Before the course](#before-the-course)
3. [Schedule](#schedule)
4. [Target group](#target-group)
5. [Learning outcomes](#learning-outcomes)
6. [Organizers and teachers](#organizers-and-teachers)

## General introduction

In metagenomics all genetic material (DNA) in a sample matrix is sequenced and analyzed. Culture-dependent methods have obvious biases in microbial ecology, while uncultured taxa comprise a large fraction of microbial communities in complex ecosystems. For this reason, metagenomics has become a widely used tool for the analysis of microbial communities. Metagenomics has also many benefits compared to amplicon sequencing, as it gives information not only on the taxonomic diversity but also on the functional composition of microbial communities. Recent bioinformatic developments now allow the recovery of metagenome-assembled genomes (MAGs) at an incredible depth.

During this course the students will learn state-of-the-art bioinformatic approaches to analyse metagenomic data. The course covers both read-based and assembly-based methods, focusing on the strenght of each of these methods depending on the research question. Short-read sequencing technologies are still commonly used in metagenomics, but recent technical advances in long-read sequencing technologies are making long-read sequencing a valid option in metagenomics. Data originating from both short- (e.g. Illumina) and long-read (e.g. Nanopore/PacBio) sequencing platforms will be discussed in this course to demonstrate their advantages and disadvantages in metagenomics and microbial ecology.

Many things are possible thanks to the support from [CSC](http://www.csc.fi).

## Before the course

Go to [myCSC](http://my.csc.fi) and join the course project (MBDP_Metagenomics). Link has been sent by email.  

__Compulsory prerequisites:__

* Basic knowledge of UNIX
* Basic knowledge of bioinformatics, molecular biosciences and microbiology

__Recommended prerequisites:__

* Basic knowledge of R programming language

## Schedule

The course will be from Monday 2nd of September to Friday 6th of September at Siltavuorenpenger 3 A, City Centre Campus.

__Practicals:__

| Day | Time | Building | Room |  
| --- | --- | --- | --- |  
| Monday 02.09.    | 10.00-17.45   | :office: __Athena__ | Hall 169  |
| Tuesday 03.09.   | 10.00-17.45   | :office: __Athena__ | Hall 169  |
| Wednesday 04.09. | 10.00-17.45   | :office: __Athena__ | Hall 169  |
| Friday 06.09.    | 10.00-17.45   | :office: __Athena__ | Hall 169  |
| Friday 15.12.    | 10.00-17.45   | :office: __Athena__ | Hall 169  |

[__-- Detailed SCHEDULE --__](Practicals/Schedule.md)  
[__-- Link to PRACTICALS --__](Practicals/README.md)

## Target group

This course is targeted to PhD students from the Doctoral Programme in Microbiology and Biotechnology (MBDP). Also other interested in microbial metagenomics are welcome.   

## Learning outcomes

By completing this course, you will:

* Have a basic understanding of metagenomic sequencing technologies and bioinformatic approaches to analyse metagenomic data
* Be able to plan and execute a metagenomic sequencing project depending on the research questions.
* Have an up-to-date knowledge on the bioinformatic tools and best practices for the analysis of metagenomes.
* Be able to choose and critically evaluate new tools and approaches for specific research question
* Have confidence to learn and implement new bioinformatic methods using available documentation

## Organizers and teachers

:woman_technologist: Henni Hultman,  Natural Resources Institute Finland (LUKE)  
:man_technologist: Antti Karkman, University of Helsinki  
:man_technologist: Freed Ahmad, University of Helsinki  
